pkgs/development/tools/misc/astyle/default.nix

```
@@ -2,11 +2,11 @@
 
 stdenv.mkDerivation rec {
   pname = "astyle";
-  version = "3.1";
+  version = "3.4.10";
 
   src = fetchurl {
-    url = "mirror://sourceforge/${pname}/${pname}_${version}_linux.tar.gz";
-    sha256 = "1ms54wcs7hg1bsywqwf2lhdfizgbk7qxc9ghasxk8i99jvwlrk6b";
+    url = "mirror://sourceforge/project/${pname}/${pname}/${pname}%203.4/${pname}-${version}.tar.bz2";
```

```suggestion
    url = "mirror://sourceforge/${pname}/${pname}-${version}.tar.bz2";
```

It looks like you can use the simpler URL here (otherwise the `3.4` should be un-hardcoded in the URL via `lib.majorMinor`, but just changing it to the shorter URL is easier)

---

Thanks! I believe in my initial testing I wasn't able to download it from the short URL, but that was probably because I was still using the old extension...

`lib.majorMinor` will be handy in the future!